gprofiler/merge.py

```diff
@@ -218,13 +219,14 @@ def _make_profile_metadata(docker_client: Optional[DockerClient]) -> str:
         container_names = []
         enabled = False
 
-    return "# " + json.dumps(
-        {
-            'containers': container_names,
-            'hostname': get_hostname(),
-            'container_names_enabled': enabled,
-        }
-    )
+    profile_metadata = {
+        'containers': container_names,
+        'container_names_enabled': enabled,
+    }
+    profile_metadata["metadata"] = metadata
+    formatted_profile_metadata = "# " + json.dumps(profile_metadata)
+    formatted_profile_metadata = formatted_profile_metadata.replace('\n', ' ')
```

Comment:
Why this?

If removing, you can step back to:
```
    return "# " + json.dumps(
        {
            'containers': container_names,
            'hostname': get_hostname(),
            'container_names_enabled': enabled,
            'metadata': metadata,
        }
    )
```

Reply:
Some of the metadata fields (like the Python version) contain `\n` so I'm removing them to make sure we won't have issues parsing them later

Reply:
What issues? `json.dumps` escapes `\n`s. `json.loads` later unescapes them